File: packages/app/src-capacitor/ios/App/App/Plugins/Echo/EchoPlugin.m
//
//  EchoPlugin.m
//  App
//
//  Created by gjm on 2022/12/30.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(EchoPlugin, "Echo",
    CAP_PLUGIN_METHOD(echo, CAPPluginReturnPromise);
)

File: packages/app/src-capacitor/ios/App/App/Plugins/Echo/EchoPlugin.swift
//
//  EchoPlugin.swift
//  App
//
//  Created by gjm on 2022/12/30.
//

import Capacitor
@objc(EchoPlugin)
public class EchoPlugin: CAPPlugin {
    @objc func echo(_ call: CAPPluginCall) {
        let value = call.getString("value") ?? ""
        call.resolve(["value": value])
    }
}
